### DISCUSSION TITLE
OpenVino backend's speed is dissapointing.

### DISCUSSION THREAD

**Discussion by zkvsky** [ggml-org/whisper.cpp#1858] [Q&A]
I've been testing whisper.cpp+OpenVino backend with GPU [Integrated Xe GPU]. And while it makes things faster [slightly lower CPU usage with maxed threads] it only uses GPU lightly [~20-40%] and in spaced out bursts.
Recently I've checked HF whisper with their optimum+openvino gpu backend, and it's the other way around. GPU is constantly maxed out,and CPU is used slightly, it's also blazing fast, orders of magnitude faster.

What's the reason behind such discrepancies in behaviour? Is it the chunking of HF?

**Comment by pharmacologic** [comment]
i just got whisper.cpp+openvino working on my own iGPU (ended up needing 'legacy' packages for the intel compute runtime w/ a 6th gen i5), i noticed the same behavior: mostly CPU, only slightly less than when otherwise, with spaced out bursts on the GPU

so i'm curious about this too

**Reply by zkvsky** [reply]
I think that they've abandoned OpenVino pretty much and they're going with ipex+SYCL instead. I've tested SYCL acceleration recently and it worked; was really annoying to setup though.
